geniza/corpus/templates/corpus/document_detail.html

```diff
@@ -21,6 +21,16 @@ <h1>{{ document.title }}</h1>
         {% comment %}Translators: Date document was first added to the PGP{% endcomment %}
         <dt class="sr-only">{% translate 'First input' %}</dt>
         <dd>{{ document.log_entries.last.action_time.year }}</dd>
+
+        {% if document.editions %}
+        {% comment %}Translators: Editor label{% endcomment %}
+        <dt>{% translate 'Editor' %}</dt>  {# optionally pluralize? #}
```

Comment:
plurals when translating are...interesting, and probably something we should tackle (or at least get exposed to) earlier rather than later. i'm fine with just using the plural always and having it be a one-element list though, either way

Reply:
yeah, I looked it up but I also wasn't sure how to reliably figure out when something had multiple editors. Multiple editions? Multiple authors on any of the editions? What if there is a single-author edition with corrections from some one else?